targets/targets.json

```diff
@@ -1801,57 +1801,42 @@
         "supported_form_factors": ["ARDUINO"],
         "release_versions": ["5"],
         "config": {
-            "usb_tx": {
+            "stdio_uart_tx": {
```

Comment:
If the old name is used, this breaks it. Shouldn't this be done backward compatible, and then cleaned up for the minor where it can be documented how to update the application?

This change could be as is, but the target code should handle both cases (thus target override in the app would work if it overrides usb_tx there)

Reply:
Yes, I agree.
I will update code.